builder/dockerfile/internals.go

```diff
@@ -137,6 +171,53 @@ func (b *Builder) performCopy(state *dispatchState, inst copyInstruction) error
 	return b.exportImage(state, imageMount, runConfigWithCommentCmd)
 }
 
+func parseChownFlag(flChown string) (string, string) {
```

Comment:
I'm surprised this doesn't exist in `pkg/idtools`. I think it should probably move to `idtools.ParseIDPair(string) (IDPair, error)`